File: python/quiz.py
s = str ( input ( ) )
s1 = s[0] + s[len(s)-1]
if int(s) % int(s1) == 0:
	print("True")
else:
	print("False")
"""
Output
121
True 
#121%11=0
Output 2
123
False
#123%13=6
"""

File: c/quiz.c
#include<stdio.h>
int main(){
char str1[] = "Codeknow";
char str2[] = {'c','o','d','e','k','n','o','w'};
int n1 = sizeof(str1) / sizeof(str1[0]);
int n2 = sizeof(str2) / sizeof(str2[0]);
printf("n1 = %d , n2 = %d",n1,n2);
return 0;
}
/*
Output:
n1 = 9 , n2 = 8
*/
